xsd/pmo/people.xsd

```diff
@@ -84,6 +84,26 @@ SOFTWARE.
           <xs:field xpath="."/>
         </xs:unique>
       </xs:element>
+      <xs:element name="badges" minOccurs="0">
```

Comment:
@amihaiemil that's the idea. Not sure this is the preferred approach but I wanted to avoid making the element mandatory. Besides, the user doesn't have badges until he reaches `1024` points.